.gitignore

```diff
@@ -123,3 +123,6 @@ lightning_logs
 *.gz
 .DS_Store
 .*_submit.py
+
+# Ignore notebooks
+notebooks/
```

Comment:
why is this needed?

Reply:
Similar to your examples/plotting.py, I created a notebooks/plotting.ipynb file to demonstrate exemplary use of the new plot methods. Since jupyter notebooks don't work well with git, I thought I would exclude the directory.
Do you have a general guideline for working with jupyter notebooks?
Otherwise I might need to manually ignore my notebooks on every commit

Reply:
Actually I realized that my jupyter notebook in the notebooks/ directory is also causing problems for building the docs, as sphinx includes the directory as source. The solution I came up with is to move my notebook into the already ignored .vscode/ directory. I will therefore revert this change.